lib/project_types/script/script_project.rb

```diff
@@ -7,11 +7,16 @@ class ScriptProject < ShopifyCli::Project
 
     attr_reader :extension_point_type, :script_name, :language
 
-    def initialize(dir)
-      super(dir)
+    def initialize(directory:)
+      super(directory: directory)
```

Comment:
```suggestion
    def initialize(*args)
      super
```
Would work too. I think thats probably the better approach because this way if ShopifyCli::Project ends up adding in extra properties this line wouldn't fail. This will just pass up the provided args to the ShopifyCli::Project initializer